frappe/public/js/frappe/list/base_list.js

```diff
@@ -199,11 +199,11 @@ frappe.views.BaseList = class BaseList {
 		if (frappe.boot.desk_settings.view_switcher && !this.meta.force_re_route_to_default_view) {
 			/* @preserve
 			for translation, don't remove
-			__("List View") __("Report View") __("Dashboard View") __("Gantt View"),
```

Comment:
Read the comment above :grimacing: It's explicitly kept like this for translation extraction. Source strings wont be extractable after you change. 


Also languages aren't that simple so `translation of view type` + `translation of "view"` != translation of entire string. 


Lets revert this change.

Reply:
@ankush, Thanks for reviewing. I think I found the problem, when I was doing the translation, I executed the 'bench get-untranslated' command that generated a list of untranslated texts that included `{0} View`, with the translation of this text caused the information in the selector to be partially translated.

As I understand it, the execution of this function looks like this: `__(__('{0} View', [this.view_name]))`, if the text `{0} View` is translated, it breaks the translation afterwards, I think it is better to replace it with `$.format('{0} View', [this.view_name])`

Without `{0} View` Translation:
![image](https://github.com/frappe/frappe/assets/16943898/76dc66c6-1288-4217-9dba-0fbf33775b6a)

With `{0} View` Translation to `Visualização em {0}`:
![image](https://github.com/frappe/frappe/assets/16943898/0d7a14ed-fb4a-4cd4-a9c1-8ceb24987493)